### PR TITLE
Master represents the future version of the spec, use the future version in the text.

### PR DESCRIPTION
**Pull request by n3wscott**
This change bumps the version used in master to `0.3-wip` of the spec.

This becomes very clear (and confusing) when you are reading the examples of features are are only in `0.3-wip`, like Batch support.

Signed-off-by: Scott Nichols <nicholss@google.com>

### CODE REVIEW

**Comment by duglin** [comment]
@n3wscott we talked about this on this week's call and there was agreement to change the version to something other than 0.2 but 0.3 didn't seem appropriate since it's not 0.3 yet.  The group seemed to like `0.3.wip` - you ok with that?

**Comment by n3wscott** [comment]
Yeah I can make that update! That works for me.

**Comment by duglin** [comment]
thanks!

File: roadmap.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ _0.2_ - Completed - 2018/12/06
 1. Define the set of protocol and serialization mappings we're going to produce
    for 0.4 milestone
 
-_0.3_
+_0.3-wip_
```
Comment (duglin) [inline roadmap.md]: I think this one needs to be skipped

Comment (n3wscott) [reply roadmap.md]: whoops!

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 4/18 call

**Pull request merged**